packages/xod-client/src/project/components/NodePinsOverlay.jsx

```diff
@@ -50,7 +50,7 @@ class NodePinsOverlay extends React.Component {
         {...size}
         className="NodePinsOverlay"
         viewBox={`0 0 ${size.width} ${size.height}`}
-        title={nodeLabel}
+        data-label={nodeLabel} // for func tests
```

Comment:
Maybe add a little prefix to indicate that this it's specifically for func tests?